modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -134,7 +131,7 @@ ComputeMultipleInelasticStress::initialSetup()
 void
 ComputeMultipleInelasticStress::computeQpStress()
 {
-  if ((isParamValid("initial_stress")) && !_elasticity_tensor_is_constant[_qp])
+  if ((isParamValid("initial_stress")) && !_is_elasticity_tensor_guaranteed_constant_in_time)
```

Comment:
This, too, should go to initial setup (the isParamValid check contains lots of slow string comparisons)